ingest/expression_files/expression_files.py

```diff
@@ -181,10 +178,85 @@ def insert(docs: List, collection_name: str, client):
                 f"Error caused by inserting into collection '{collection_name}': {e}"
             )
 
-    def load(self, gene_docs: List, data_array_docs: List):
+    def load(self, docs: List, collection_name: List):
         start_time = datetime.datetime.now()
-        GeneExpression.insert(gene_docs, self.COLLECTION_NAME, self.mongo_connection)
-        GeneExpression.insert(data_array_docs, "data_arrays", self.mongo_connection)
+        GeneExpression.insert(docs, collection_name, self.mongo_connection)
         self.info_logger.info(
-            f"Time to load {len(gene_docs) + len(data_array_docs)} models: {str(datetime.datetime.now() - start_time)}"
+            f"Time to load {len(docs)} models: {str(datetime.datetime.now() - start_time)}"
         )
+
+    def create_models(
```

Comment:
@eweitz This is the function that @devonbush was referring to and you had touched on in this [comment](https://github.com/broadinstitute/scp-ingest-pipeline/pull/132#discussion_r475696783).